h/models/subscriptions.py

```diff
@@ -26,9 +37,5 @@ class Subscriptions(Base):
     active = sa.Column(sa.Boolean, default=True, nullable=False)
     """Whether the subscription is active or not."""
 
-    @classmethod
-    def get_subscriptions_for_uri(cls, session, uri):
-        return session.query(cls).filter(func.lower(cls.uri) == func.lower(uri)).all()
```

Comment:
While it's horrible to have this here, it's certainly a lot less boiler plate than creating a service.